aria/cli/commands.py

```diff
@@ -241,12 +241,9 @@ def _get_workflow(self, context, workflow_name):
             if workflow.type.role != 'workflow':
                 raise AttributeError('policy is not a workflow: "{0}"'.format(workflow_name))
 
-            try:
-                sys.path.append(workflow.properties['implementation'].value)
-            except KeyError:
-                pass
+            sys.path.append(os.path.dirname(str(context.presentation.location)))
```

Comment:
My previous implementation had a stupid extra `function` property which was just a temporary solution. Obviously it's enough to just have an `implementation` property. Until we handle true CSAR, I guess we can just pretend that artifacts are in the same directory as the main YAML.